controllers/feeds/getFeedsV2.js

```diff
@@ -155,14 +156,16 @@ module.exports = async (req, res) => {
       listBlockDomain,
       listPostAnonymousAuthor,
       isBlurredPost,
-      allFollowingUser
+      allFollowingUser,
+      anonymUserId
     ] = await Promise.all([
       UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
       getListBlockUser(req.userId),
       getBlockDomain(req.userId),
       getListBlockPostAnonymousAuthor(req.userId),
       UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId),
-      UserFollowUserFunction.getAllFollowingUser(UserFollowUser, req.userId)
+      UserFollowUserFunction.getAllFollowingUser(UserFollowUser, req.userId),
+      getAnonymUser(req.userId)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `Promise.all` is a good practice for parallelizing asynchronous operations. However, it's important to note that if any promise in the array fails, `Promise.all` will immediately reject with the reason of the first promise that rejected. This could potentially stop other promises that did not yet have the chance to complete. If you want all promises to complete regardless of whether some failed or not, consider using `Promise.allSettled` instead.

```diff
-      ] = await Promise.all([
+      ] = await Promise.allSettled([
         UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
         getListBlockUser(req.userId),
         getBlockDomain(req.userId),
         getListBlockPostAnonymousAuthor(req.userId),
         UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId),
        UserFollowUserFunction.getAllFollowingUser(UserFollowUser, req.userId),
        getAnonymUser(req.userId)
     ]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->